dashboard/test/ui/features/teacher_tools/teacher_dashboard/teacher_dashboard_progress_v2.feature

```diff
@@ -28,6 +28,52 @@ Scenario: Teacher can open and close Icon Key and details
   And I click selector "#ui-close-dialog"
   And element "h3:contains(Progress Tracking Icon Key)" is hidden
 
+Scenario: Viewing student metadata
+  Given I create an authorized teacher-associated student named "Sally"
+  Given I am assigned to unit "allthethings"
+  And I am on "http://studio.code.org/s/allthethings/lessons/44/levels/9?noautoplay=true"
+  And I wait to see "#runButton"
+  When I press "runButton"
+  And I wait for 5 seconds
+  And I submit this level
+
+  # Progress tab
+  When I sign in as "Teacher_Sally" and go home
+  And I get levelbuilder access
+  And I navigate to the V2 progress dashboard for "Untitled Section"
+
+  # Toggle to V2 progress view
+  And I wait until element "h6:contains(Icon Key)" is visible
+  And I wait until element "#ui-test-progress-table-v2" is visible
+
+  # Can see and open menu with more options
+  And I wait until element "#ui-see-more-options-dropdown" is visible
+  Then I click selector "#ui-see-more-options-dropdown"
+  And I wait until element "div:contains(Expand all student rows)" is visible
+  And I wait until element "div:contains(Collapse all student rows)" is visible
```

Comment:
Nit: this doesn't need to be wait until. Both will load at the same time so the second can is just `and _ is visible` or whatever it is.

Reply:
Ugh... this is where I feel like I get nervous - I know "technically" we don't need the "wait until" BUT it seems like sometimes what causes some flaky tests is inconsistent loading (thus my preference to use "wait until" more often than not).  I am wondering if you see any harm in defaulting to it - thoughts?